src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -1,55 +1,125 @@
-import React, { useCallback, useState } from "react";
+import React, { useCallback } from "react";
 import { Modal } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+import { MAX_TABLE_CAPACITY, MIN_TABLE_CAPACITY } from "settings";
+
+import { InputField } from "components/atoms/InputField";
 
 import "./EditTableTitleModal.scss";
 
 export interface EditTableTitleModalProps {
   isShown: boolean;
   title: string;
+  description?: string;
+  capacity: number;
   error: string;
   onHide: () => void;
   onCancel: () => void;
-  onSave: (newTableTitle: string) => void;
+  onSave: (values: EditTableForm) => void;
+}
+
+export interface EditTableForm {
+  title: string;
+  description?: string;
+  capacity: number;
 }
 
 export const EditTableTitleModal: React.FC<EditTableTitleModalProps> = ({
   isShown,
   title,
+  description,
+  capacity,
   error,
   onSave,
   onHide,
   onCancel,
 }) => {
-  const [newTableTitle, setNewTableTitle] = useState<string>(title);
+  const {
+    register,
+    handleSubmit,
+    errors,
+    // formState,
+    // setError,
+    // clearError,
+    watch,
+  } = useForm<EditTableForm>({
+    mode: "onChange",
+    reValidateMode: "onChange",
+    defaultValues: {
+      title,
+      description,
+      capacity,
+    },
+  });
 
-  const saveChanges = useCallback(() => {
-    onSave(newTableTitle);
-    onHide();
-  }, [newTableTitle, onHide, onSave]);
+  const values = watch();
 
-  const changeTableTitle = useCallback(
-    (e: React.ChangeEvent<HTMLInputElement>) => {
-      setNewTableTitle(e.target.value);
-    },
-    []
-  );
+  const onSubmit = useCallback(() => {
+    onSave(values);
+    onHide();
+  }, [onHide, onSave, values]);
 
   return (
     <Modal show={isShown} onHide={onHide}>
       <Modal.Body>
-        <input value={newTableTitle} onChange={changeTableTitle}></input>
-        {error && <label className="input-error">{error}</label>}
-        <div className="edit-title-buttons">
-          <button className="btn btn-centered btn-secondary" onClick={onCancel}>
-            Cancel
-          </button>
-          <button
-            className="btn btn-centered btn-primary"
-            onClick={saveChanges}
-          >
-            Save Changes
-          </button>
-        </div>
+        <form onSubmit={handleSubmit(onSubmit)} className="edit-table-modal">
+          <InputField
+            ref={register({ required: true })}
+            name="title"
+            containerClassName="edit-table-modal__input--spacing"
+            placeholder="Table topic"
+          />
+          {errors.title && errors.title.type === "required" && (
+            <span className="input-error">Title is required</span>
+          )}
+          <InputField
```

Comment:
[nit] Spacing between 'different chunks' in the JSX.

```suggestion

          <InputField
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a